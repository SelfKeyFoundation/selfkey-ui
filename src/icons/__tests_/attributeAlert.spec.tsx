import * as React from 'react';
import { create } from 'react-test-renderer';
import AttributeAlertIcon from '../attribute-alert';

describe('Attribute Alert Icon', () => {
  it('should show Attribute Alert Icon', () => {
    const component = create(
      <AttributeAlertIcon />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});