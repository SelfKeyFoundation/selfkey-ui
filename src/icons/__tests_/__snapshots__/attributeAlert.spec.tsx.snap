// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Attribute Alert Icon should show Attribute Alert Icon 1`] = `
<svg
  aria-hidden="true"
  className="MuiSvgIcon-root-1 Component-root-0-1-1"
  focusable="false"
  role="presentation"
  version="1.1"
  viewBox="0 0 18 20"
>
  <g
    fill="none"
    fill-rule="evenodd"
    id="ðŸ’¹-Marketplace"
    stroke="none"
    stroke-width="1"
  >
    <g
      fill="#E98548"
      id="703.1---Incorporation-+-Submit-+-Errors"
      transform="translate(-385.000000, -635.000000)"
    >
      <g
        id="Modal1"
        transform="translate(330.000000, 159.000000)"
      >
        <g
          id="Table"
          transform="translate(40.000000, 264.000000)"
        >
          <g
            id="Col0"
            transform="translate(15.000000, 55.000000)"
          >
            <path
              d="M17.2495253,159.852929 C17.2767483,159.96155 17.290502,160.073107 17.2904765,160.185088 L17.2904765,166.236755 C17.2832562,170.009674 15.2457557,173.486638 11.9577295,175.337005 L9.31410666,176.824896 C8.89875475,177.058368 8.39172171,177.058368 7.9763698,176.824896 L5.33274695,175.337005 C2.04472075,173.486638 0.00722028608,170.009674 -1.63424829e-13,166.236755 L-1.63424829e-13,160.185088 C-1.63424829e-13,159.875679 0.122853385,158.82915 1.66534589,158.055629 C3.06223438,157.35491 5.41009908,157 8.64523823,157 C15.5113774,157 16.9810679,158.788199 17.2495253,159.852929 Z M15.9254388,166.223104 L15.9254388,160.185088 C15.9254388,160.185088 15.4704263,158.365038 8.64523823,158.365038 C1.36503762,158.365038 1.36503762,160.185088 1.36503762,160.185088 L1.36503762,166.236755 C1.36484432,169.523732 3.13720806,172.555361 6.00161538,174.167623 L8.64523823,175.655514 L11.2888611,174.153973 C14.1532684,172.54171 15.9256321,169.510081 15.9254388,166.223104 Z M8.41773196,172.470426 C8.04078723,172.470426 7.73521315,172.164852 7.73521315,171.787907 C7.73521315,171.410963 8.04078723,171.105389 8.41773196,171.105389 C8.79467669,171.105389 9.10025077,171.410963 9.10025077,171.787907 C9.10025077,172.164852 8.79467669,172.470426 8.41773196,172.470426 Z M8.41773196,160.867607 C8.79467669,160.867607 9.10025077,161.173181 9.10025077,161.550125 L9.10025077,168.830326 C9.10025077,169.207271 8.79467669,169.512845 8.41773196,169.512845 C8.04078723,169.512845 7.73521315,169.207271 7.73521315,168.830326 L7.73521315,161.550125 C7.73521315,161.173181 8.04078723,160.867607 8.41773196,160.867607 Z"
              id="icon-attribute-alert"
            />
          </g>
        </g>
      </g>
    </g>
  </g>
</svg>
`;
